config/settings/development.yml

```diff
@@ -0,0 +1,36 @@
+:log:
+  :collection:
+    :archive:
+      :pattern:
+    :current:
+      :pattern:
+      - log/*.log
+      - log/apache/*.log
+      - log/*.txt
+      - config/*
+      - "/var/log/syslog*"
+      - "/var/log/daemon.log*"
+      - "/etc/default/ntp*"
+      - "/var/log/messages*"
+      - "/var/log/cron*"
+      - BUILD
+      - GUID
+      - VERSION
+    :ping_depot: true
+    :ping_depot_timeout: 20
+  :level: debug
+  :level_rails: debug
```

Comment:
You don't need the entire subsection...you can just put the one key-path that is different.